src/ui/views/ViewGetHelp.qml

```diff
@@ -107,7 +108,7 @@ Item {
             imageLeftSrc: "qrc:/ui/resources/developer.svg"
             imageRightSrc: "qrc:/nebula/resources/chevron.svg"
             visible: VPNSettings.developerUnlock
-            onClicked: mainStackView.push("qrc:/ui/developerMenu/ViewDeveloperMenu.qml")
+            onClicked: VPN.state === VPN.StateMain ? settingsStack.push("qrc:/ui/developerMenu/ViewDeveloperMenu.qml") : mainStackView.push("qrc:/ui/developerMenu/ViewDeveloperMenu.qml")
```

Comment:
What happens if we continue pushing this exclusively to `mainStackView` ?

Reply:
It would be pushed on top of StateMain which holds all the nav bar stacks, so you'd still see the nav bar, but you wouldn't see any of the changes happen when clicking a nav bar button because it'd behind the dev menu